database/migrations/2023_02_04_192730_create_payments_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('payments', function (Blueprint $table) {
+            
+            $table->integer('userID');
+            $table->integer('productID');
+            $table->integer('quantity');
+            $table->tinyInteger('status');
+            $table->timestamps();
```

Comment:
đối với timestamp nên có 1 giá trị default
```suggestion              
            $table->timestamp('created_at')->default(now()->toDateTimeString());
            $table->timestamp('updated_at')->default(now()->toDateTimeString());
```